### ISSUE TITLE
Error with converting any images

### ISSUE THREAD

**Issue by creepaminer** [pmrowla/pylivemaker#127]
* pylivemaker version: v1.1.2
* Python version: 3.10.6
* Operating System: Windows 10
* LiveMaker game: (No sorry its nsfw)

### Description

I am trying to extract all of this game's assets to see if I can use some for another project but I'm running into issues when trying to convert any image to a readable format. The game extracts all files just fine but I'm only running into this error with the .gal files.

This game says it's from 2010, and I read someone saying that the engine wasn't even native to Windows 7 yet, so I'm wondering if this is some old format in the gal files that is causing the conversion issues.

### What I Did

When I run ```galconvert image.gal image.png``` I get the error message below
```
Traceback (most recent call last):
  File "C:\Users\User\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\User\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\User\AppData\Local\Programs\Python\Python310\Scripts\galconvert.exe\__main__.py", line 7, in <module>
  File "C:\Users\User\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "C:\Users\User\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "C:\Users\User\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "C:\Users\User\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "C:\Users\User\AppData\Local\Programs\Python\Python310\lib\site-packages\livemaker\cli\cli.py", line 62, in galconvert
    im = Image.open(input_file)
  File "C:\Users\User\AppData\Local\Programs\Python\Python310\lib\site-packages\PIL\Image.py", line 3288, in open
    im = _open_core(fp, filename, prefix, formats)
  File "C:\Users\User\AppData\Local\Programs\Python\Python310\lib\site-packages\PIL\Image.py", line 3274, in _open_core
    im = factory(fp, filename)
  File "C:\Users\User\AppData\Local\Programs\Python\Python310\lib\site-packages\PIL\ImageFile.py", line 137, in __init__
    self._open()
  File "C:\Users\User\AppData\Local\Programs\Python\Python310\lib\site-packages\livemaker\GalImagePlugin.py", line 306, in _open
    self.seek(0)
  File "C:\Users\User\AppData\Local\Programs\Python\Python310\lib\site-packages\livemaker\GalImagePlugin.py", line 322, in seek
    self.mode = mode
AttributeError: can't set attribute 'mode'
```

I get the exact same error message when I try to extract the game while converting images at the same time using 
```lmar x -i png game.exe -o game_files```
Except this time when I use this command, when it outputs files it doesn't output anything at all to the folder other than the bunch of 00000000.lsb files as well as the live.lpb and thats it.

I can provide a few example gal files if its needed

**Comment by pmrowla** [comment]
Fixed by ade58b276cedc6858716d1f3e6109f875dd79905

pylivemaker 1.2.0 was just released and should fix this issue for you (when using either `galconvert` or `lmar x -i`)

**Comment by creepaminer** [comment]
Ah a super simple fix then, I was overcomplicating my assumption haha. Anyway, it works like a charm now thanks!
